docs/wallets/cli/dcrwallet-tickets.md

```diff
@@ -54,148 +50,63 @@ Once `dcrwallet` is restarted with that line in `dcrwallet.conf`, your wallet wi
-
-    A brief overview of configuring your VSP is provided here:
-
-    1. With your wallet open, issue the `dcrctl --wallet getnewaddress` command to retrieve an address.
-
-    1. Using that address, issue the `dcrctl --wallet validateaddress <address from step 1>` command. This should return a JSON object that will be displayed like so:
-
-            {
-              "isvalid": true,
-              "address": "DsExampleAddr1For2Demo3PurposesOnly",
-              "ismine": true,
-              "pubkeyaddr": "DkExample0Addr1For2Demo4Purposes5Only6Do7Not8Use9Pls0",
-              "pubkey": "022801337beefc0ffee1dab8d4ffa898a782466c9a1fc00ca8863de5438dc07dcc",
-              "iscompressed": true,
-              "account": "voting"
-            }
-
-    1. Copy the `pubkeyaddr` into the VSP's "Submit Address" form and hit the submit button. The page should redirect you to the tickets page, which will display more instructions.
-
-    1. At the top of the tickets page, you should see a "Ticket Information" section. Copy your "Redeem Script" and use it to issue the `dcrctl --wallet importscript <Insert Redeem Script Here>` command.
-
-    With the VSP configured and your multi-sig script imported to your wallet, you can now start [purchasing tickets](#ticket-purchasing).
-
----
-
-## Ticket Purchasing
-
-Both manual and automatic ticket purchasing require your wallet to be unlocked via the `promptsecret | dcrctl --wallet walletpassphrase - <time limit>` command.
-
-There are three things you might want to understand before purchasing tickets: the `purchaseticket` command, when/why a `ticketfee` is important, and the significance of `ticket price`.
-
-##### `purchaseticket` Command
+```ini
+; ------------------------------------------------------------------------------
+; VSP settings
+; ------------------------------------------------------------------------------
 
-The `purchaseticket` command will be used to purchase tickets whether manual or automatic. Let's take a closer look at the command and its arguments:
+; The URL of the VSP.
+vsp.url=https://teststakepool.decred.org
 
-```no-highlight
-purchaseticket "fromaccount" spendlimit (minconf=1 "ticketaddress" numtickets "pooladdress" poolfees expiry "comment" ticketfee)
+; The base64 encoded public key of the VSP server.  This can be found on the
+; VSP website in the footer.
+vsp.pubkey=ia9Ra2Drb+OHLqRyBsJnRKBd7TUG1IvrseC6robKzGo=
 ```
 
-1. `fromaccount`    =  Required String: The account from which to purchase tickets (e.g. "default").
-1. `spendlimit`     =  Required Number: Limit on the amount to spend on ticket (e.g. 50).
-1. `minconf`        =  Optional Number: Minimum number of block confirmations required (e.g. 1).
-1. `ticketaddress`  =  Optional String: The ticket address to which voting rights are given
-1. `numtickets`     =  Optional Number: The number of tickets to purchase at once (e.g. 1)
```

Comment:
`numtickets` is still useful for purchasing multiple tickets at once. But to use it you have to also satisfy `minconf` and `ticketaddress` (emtpy string for VSPv3) params AND it doesn't always work as expected (see other comments) so not sure if we want to mention it here, omit, or a middleground "run `dcrctl --wallet help purchaseticket` for advanced options".

Reply:
Added the help command so advanced users can explore it themselves.